static/js/rich_search/app.tsx

```diff
@@ -131,14 +151,15 @@ function App({
 async function getStatVars(
   dcids: string[],
   query: string
-): Promise<NamedNode[]> {
+): Promise<[string[], { [key: string]: StatVarInfo }]> {
   const params = new URLSearchParams({ query });
   for (const place of dcids) {
     params.append("place", place);
   }
   return axios
     .get(`/api/stats/stat-var-search-ai?${params.toString()}`)
-    .then((resp) => resp.data.statVars || []);
+    .then((resp) => (resp.data.statVars || []).map((v) => v.dcid).slice(0, 6))
```

Comment:
are dcid's guaranteed in the response?

Reply:
They are, but I added a filter just in case